package com.imooc.test;

public class TryCatchTest {

	public static void main(String[] args) {
		TryCatchTest t = new TryCatchTest();
		int result = t.test();
		System.out.println("test()方法执行完毕，返回值为"+result);

	}
	
	public int test() {
		int divider=10;
		int result = 100;
		
		try {
			while(divider>-1){
				divider--;
				result = result+100/divider;
			}
			return result;
		} catch (Exception e) {
			e.printStackTrace();
			System.out.println("抛出异常了！");
			return -1;
		}
	}
	
	public int test2() {
		return 0;
		
	}

}
